client/my-sites/plans-v2/more-info-box/style.scss

```diff
@@ -0,0 +1,20 @@
+.more-info-box__more-container {
+	background-color: rgba( 157, 217, 119, 0.1 );
```

Comment:
It's best practice to use https://color-studio.blog/ colour. I believe you can use `transparentize` in sass to play with the opacity.